src/src/pages/capabilities/KafkaCluster/index.js

```diff
@@ -92,7 +92,7 @@ export default function KafkaCluster({ cluster, capabilityId }) {
 
   return (
     <PageSection
-      headline={`Kafka Topics (${cluster.name.toLocaleLowerCase()})`}
+      headline={` `} headlineChildren={(<div style={{display: 'flex', alignItems: 'center'}}><span style={{}}>Kafka Topics ({cluster.name.toLocaleLowerCase()}  -&nbsp;</span><span style={{fontSize: '0.8em', color: '#c6c7c5', top: '1px', position: 'relative'}}>{cluster.id}</span>)</div>)}
```

Comment:
Could put the styling into its own separate file. Dunno.

Reply:
please do :)

Reply:
I believe having the style in the same file is preferable (and the react-way), but it should also preferably not be inline